### PR TITLE
Additional cucumbers

### PR DESCRIPTION
**Pull request by BenHand**
testing joining a meeting and results that are expected on certain situations

### CODE REVIEW

**Comment by jah2488** [comment]
:+1:

File: app/controllers/meetings_controller.rb [hunk 1]
```diff
@@ -63,7 +63,7 @@ def join
         em.save
         message = {notice: 'Employee successfully joined!'}
       else
-        message = {alert: 'You own or are already in another meeting at this time.'}
+        message = {alert: 'You own  are already in another meeting at this time.'}
```
Comment (JMD1986) [inline app/controllers/meetings_controller.rb]: why did you remove the or?

Comment (mlgiardina) [inline app/controllers/meetings_controller.rb]: why did you delete the 'or' ?

**Comment by JMD1986** [comment]
:+1:

File: app/controllers/meetings_controller.rb (hunk 1 above)
Comment (jah2488) [inline app/controllers/meetings_controller.rb]: why did you expunge the 'or' ?
